I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 820, in script
    $ renpy.pause(1.5)
  File "game/script.rpy", line 820, in <module>
    $ renpy.pause(1.5)
TypeError: missing a required argument: 'g_score'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 820, in script
    $ renpy.pause(1.5)
  File "/Applications/renpy-8.3.2-sdk/renpy/ast.py", line 821, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-8.3.2-sdk/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 820, in <module>
    $ renpy.pause(1.5)
  File "/Applications/renpy-8.3.2-sdk/renpy/exports/statementexports.py", line 194, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay, pause_modal=modal)
  File "/Applications/renpy-8.3.2-sdk/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-8.3.2-sdk/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Applications/renpy-8.3.2-sdk/renpy/display/core.py", line 2745, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Applications/renpy-8.3.2-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-8.3.2-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-8.3.2-sdk/renpy/display/displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-8.3.2-sdk/renpy/display/screen.py", line 480, in visit_all
    callback(self)
  File "/Applications/renpy-8.3.2-sdk/renpy/display/core.py", line 2745, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Applications/renpy-8.3.2-sdk/renpy/display/screen.py", line 491, in per_interact
    self.update()
  File "/Applications/renpy-8.3.2-sdk/renpy/display/screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "/Applications/renpy-8.3.2-sdk/renpy/parameter.py", line 455, in apply_arguments
    return parameters.apply(args or (), kwargs or {}, ignore_errors)
  File "/Applications/renpy-8.3.2-sdk/renpy/parameter.py", line 307, in apply
    raise TypeError(msg) # from None
TypeError: missing a required argument: 'g_score'

macOS-14.0-arm64-arm-64bit arm64
Ren'Py 8.3.2.24090902
star team 1.0
Mon Nov 18 00:11:08 2024
